CanvasColorSpaceProposal.md

```diff
@@ -139,14 +138,23 @@ else
 
 #### WebGL behavior
 
-* Values stored in WebGL backbuffers are in the canvas' color space.
-* Values written by ``gl_FragColor`` use the primaries of the canvas' color space.
-* The color encoding function is defined as follows:
-* For formats of ``"unorm8"``, ``"unorm16"``, and ``"float16"`` the color encoding function is:
+Values stored in WebGL backbuffers are in the canvas' color space.
+This includes values accessed directly (e.g, through `ReadPixels`) and all values written via shaders in the graphics pipeline.
+
+The context attribute ``framebufferColorEncodingSRGB`` controls the color encoding function applied to values assigned to ``gl_FragColor`` after the blending stage of the graphics pipeline.
```

Comment:
Let's take the opportunity to correct this terminology since ESSL 300 shaders (used by WebGL 2.0) have user-defined fragment shader output variables.

"...applied to the fragment shader's color output value..."

Not sure whether it's necessary to mention again here that this is only done when writing to WebGL's built-in back buffer, rather than user-defined framebuffers.

Reply:
Done. Added "when writing to the WebGL default back buffer"

Reply:
Done